tutorials/install.md

```diff
@@ -17,4 +17,4 @@ Installing WhoWroteThat
 
 ## As a browser extension
 
-1. Install it. (Details t.b.c.)
+Refer to [the README file](../README.md).
```

Comment:
Eventually this would be a link to entry on the browser's extension marketplace (Google Web Store, etc.), not how to build it from source.

Reply:
Reverted this for now, since this probably won't be released soon on the web store.